unified-codes/apps/legacy-api/src/app/api.ts

```diff
@@ -0,0 +1,23 @@
+import fastify, { FastifyServerOptions, RouteShorthandOptionsWithHandler } from 'fastify';
+
+import schemas from './schemas';
+import handlers from './handlers';
+
+import { Server, Route, Options } from './types';
+
+export const createServer = (routes: Route[], opts?: Options) => {
+  const server: Server = fastify(opts);
+  routes.forEach((route) => server.get(route.path, route.opts));
+  return server;
+};
+
+export const createLegacyApiServer = (opts?: Options) => {
```

Comment:
Same as above, just makes testing a bit cleaner (although in this instance it might be unnecessary!).